web/src/app/modules/views/side/modules/links-actions/components/links-actions.component.ts

```diff
@@ -28,12 +28,18 @@ export class LinksActions {
         return this.additionalInformationService.element;
     }
 
-    public get model(): IContainer {
-        return this.additionalInformationService.model;
+     public get_model(): IContainer {
+        return this._model;
+    }
+    public set model(value: IContainer) {
+        this._model = value;
     }
 
     public get requirement(): Requirement {
-        return this.additionalInformationService.requirement;
+        return this._requirement;

```

Comment:
Same here. The additionalInformationService should be used to get the requirement...